src/dateutil/_version.py

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
```

Comment:
I think we need to modify `.gitignore` now that this file has moved.

Reply:
ups. let me fix that indeed.